platform/lib/routers/packager.js

```diff
@@ -43,7 +43,12 @@ const packager = (request, response, next) => {
     return;
   }
   // Don't package non valid AMP pages
-  if (!request.path.endsWith('.amp.html')) {
+  let pagesHost = config.hosts.platform.host;
+  if (config.hosts.platform.port) {
+    pagesHost += `:${config.hosts.platform.port}`;
+  }
+  if (request.get('host') !== pagesHost) {
```

Comment:
Oh I guess you we're asking to make sure the packager doesn't break on non-valid pages... How robust are the transformers of the packager?

Are samples using an experimental component for example are considered valid? Or would that break?

Reply:
Transformations are the same as AMP Optimizer (and AMP Cache). Sending non-valid pages to the packager means just more overhead.